dowhy/causal_refuters/placebo_treatment_refuter.py

```diff
@@ -65,8 +65,17 @@ def refute_estimate(self):
             new_estimator = self.get_estimator_object(new_data, identified_estimand, self._estimate)
             new_effect = new_estimator.estimate_effect()
             sample_estimates[index] = new_effect.value
+        
 
         refute = CausalRefutation(self._estimate.value, 
                                   np.mean(sample_estimates),
                                   refutation_type="Refute: Use a Placebo Treatment")
+                                  
+        # Note: We hardcode the estimate value to ZERO as we want to check if it falls in the distribution of the refuter
+        # Ideally we should expect that ZERO should fall in the distribution of the refuter as we have severed any causal
```

Comment:
"zero should fall in the distribution of the effect estimates"

Reply:
Done.